pytorch_lightning/plugins/apex.py

```diff
@@ -23,10 +23,12 @@ class ApexPlugin:
     def __init__(self, trainer):
         self.trainer = trainer
 
-    def connect(self, model, optimizers):
+    def connect(self, model, optimizers=None):
         model, optimizers = self.configure_apex(model, optimizers, self.trainer.amp_level)
```

Comment:
is `LightningModule.configure_apex` not used anymore??

Reply:
probably moved due to refactoring, but it still stays

Reply:
@williamFalcon